specs/core/1_custody-game.md

```diff
@@ -358,8 +377,7 @@ def process_bit_challenge(state: BeaconState,
     assert verify_standalone_attestation(state, convert_to_standalone(state, challenge.attestation))
     # Verify the attestation is eligible for challenging
     responder = state.validator_registry[challenge.responder_index]
-    min_challengeable_epoch = responder.exit_epoch - EPOCHS_PER_CUSTODY_PERIOD * (1 + responder.max_reveal_lateness)
-    assert min_challengeable_epoch <= slot_to_epoch(challenge.attestation.data.slot) 
+    assert responder.latest_custody_reveal_period > epoch_to_custody_period(slot_to_epoch(challenge.attestation.data.slot))
```

Comment:
I think in the current implementation there is no time limit on bit challenges. Do we want to set this to 2*CUSTODY_RESPONSE_DEADLINE ? (That means after the custody reveal there is at least CUSTODY_RESPONSE_DEADLINE to post a bit challenge)